### ISSUE TITLE
Looking for new maintainer(s)?

### ISSUE THREAD

**Issue by joesanford** [basho/riak-python-client#535]
I have been using this client in production for some time now, and have noticed that there seems to still be issues, releases, and more to be handled. I know that Basho is no more, so I was wondering if you are looking for anyone to help with maintaining the repo. I would gladly help with this project, please just let me know!

**Comment by martincox** [comment]
Absolutely. There's a myriad of clients with no one actively maintaining them.

The more help on this better, because I foresee a bunch of libraries being deprecated without someone with an interest in keeping it afloat. New features have been added into Riak, but not all of the clients, because it isn't in everyone's interest in doing so.

This is something that should be discussed at the upcoming Riak user meetup, I think. I would rationalise the client libraries unless anyone is stepping forward to actively maintain it.

**Comment by glassresistor** [comment]
This is good to know.  I'm happy to help with this.  When in the Riak users meeting? @martincox

**Comment by martincox** [comment]
It's taking place next Thursday, in Manchester, England. Everyone is welcome, so if you are around that area, feel free to come along.

**Comment by martincox** [comment]
There may be some remote arrangements for anyone who can't be there physically but still want to participate.

**Comment by joesanford** [comment]
If there is are any remote arrangements please let me know, or know where to check for them. I live in the states so unfortunately would not be able to make it.

If remote arrangements aren't around (please don't go out of your way to get them for me, I may be busy during that time anyway), please pass on the message that I would love to help.

**Comment by thedrow** [comment]
This is a shame.
If this client won't be updated we will have to drop Riak support in Celery 4.
